docs/execution_providers/Nuphar-ExecutionProvider.md

```diff
@@ -63,6 +63,22 @@ create_shared.sh -c /path/to/jit/cache/NUPHAR_CACHE_VERSION [-m optional_model_f
 # run Nuphar inference again with cached JIT dll
 ```
 
+### NGEMM
+NGEMM (Nuphar GEMM) is an optimized low-precision GEMM implementation based on compiler techniques.
+Please refer to our paper for more details of NGEMM: ["NGEMM: Optimizing GEMM for Deep Learning via Compiler-based Techniques"](https://arxiv.org/abs/1910.00178).
+
+#### NGEMM Tiling / Permutation Configuration
+NGEMM has default tiling parameters, but users can overwrite them through environment variables:
+* NUPHAR_IGEMM_TILE_M / NUPHAR_IGEMM_TILE_N / NUPHAR_IGEMM_TILE_K
+
+    These 3 parameters are the tiling sizes for corresponding dimensions of GEMM ([M x K] x [K x N]).
```

Comment:
typo: "corresponding" -> "the corresponding"

Reply:
Not a typo. the + plural count nouns are used for specific reference, cannot used for general reference "corresponding dimensions" here.

Reply:
Not want to be too pedantic, but "of GEMM" requires us to add "the" in front of corresponding. I think we need a native English speaker to help us :)